### PR TITLE
VIH-7803 staff member display name in participant waiting room

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7803

### Change description ###

- was had typo in participant waiting room

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by robertbennett1998 — DISMISSED** [review]
LGTM

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ describe('JudgeParticipantStatusListComponent', () => {
 
         expect(component.staffMembers).toBeDefined();
         expect(component.staffMembers.length).toBe(1);
+        expect(component.staffMembers[0].display_name).toBe('Staff Member');
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts:86]: can you change this to a const please, we should stop testing text in template files

Comment (PrasannaDommalapati) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts:86]: this is the hardcoded constant in the test data

Comment (PrasannaDommalapati) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts:86]: not testing text it is test data.

**Review by omio-ma — APPROVED** [review]

**Review by omio-ma — DISMISSED** [review]

**Review by FarronF — DISMISSED** [review]

**Review by omio-ma — APPROVED** [review]

**Review by robertbennett1998 — APPROVED** [review]
LGTM

**Pull request merged**